pipeline_dp/combiners.py

```diff
@@ -97,3 +99,64 @@ def merge_accumulators(self, count1: int, count2: int):
     def compute_metrics(self, count: int) -> float:
         return dp_computations.compute_dp_count(count,
                                                 self._params.mean_var_params)
+
+
+class SumCombiner(Combiner):
+    """Combiner for computing dp sum.
+
+    the type of the accumulator is int, which represents sum of the elements
+    in the dataset for which this accumulator is computed.
+    """
+    AccumulatorType = int
+
+    def __init__(self, params: CombinerParams):
+        self._params = params
+
+    def create_accumulator(self, values: Iterable[int]) -> AccumulatorType:
+        return np.clip(values, self._params.aggregate_params.low,
+                       self._params.aggregate_params.high).sum()
+
+    def merge_accumulators(self, sum1: AccumulatorType, sum2: AccumulatorType):
+        return sum1 + sum2
+
+    def compute_metrics(self, sum: AccumulatorType) -> float:
+        return dp_computations.compute_dp_sum(sum, self._params.mean_var_params)
+
+
+class CompoundCombiner(Combiner):
+    """Combiner for computing a group of dp aggregations.
+
```

Comment:
Please add something like

CompoundCombiner contains one or more combiners of other types for computing multiple metrics.
For example it can contain [CountCombiner,  SumCombiner].
CompoundCombiner delegates all operations to the internal combiners.